babel.config.js

```diff
@@ -0,0 +1,6 @@
+const BABEL_ENV = process.env.BABEL_ENV;
+const cjs = BABEL_ENV === 'cjs';
+
+module.exports = {
+  presets: ['@babel/preset-env', '@babel/preset-typescript', '@babel/preset-react'],
```

Comment:
Due to a bug (?) in preconstruct we need `plugins: ["@babel/plugin-transform-runtime"]` here else the `async await` syntax will be erroneous in the eventual output build

Reply:
Hmmmm, that's interesting. I don't think any of the `preconstruct` projects I work on use this and I've not noticed any issues with `async`/`await`